### DISCUSSION TITLE
"filter_out_message_when " and "expires_after " ignored

### DISCUSSION THREAD

**Discussion by kevinkey619** [bellrichm/WeeWX-MQTTSubscribe#160] [General]
Despite the below settings in my weewx.conf, I keep getting random blank pressure values sent to Weather Underground.  

[[[indoor/BME280_pressure]]]
            # The WeeWX name.
            # Default is the name from MQTT.
            name = pressure
            ignore = false
            qos = 1
            expires_after = None
            filter_out_message_when = 0, None

**Comment by bellrichm** [comment]
‘filter_out_message_when’ is only valid for json payloads

**Comment by bellrichm** [comment]
Lets work through your problems in one place, https://github.com/bellrichm/WeeWX-MQTTSubscribe/discussions/157
